StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/RoundedIconView.swift

```diff
@@ -46,10 +49,10 @@ final class RoundedIconView: UIView {
         switch image {
         case .image(let image):
             iconImageView.image = image.makeImage(template: true)
-        case .imageUrl(let imageUrl):
+        case .imageUrl(let imageUrl, let fallbackImage):
             iconImageView.setImage(
                 with: imageUrl,
-                placeholder: nil,
+                placeholder: fallbackImage?.makeImage(template: true),
```

Comment:
note on consistency, right now this is introducing two different namings:
1) "fallback" image
2) "placeholder" image

is there a difference? if not, then it would make sense that both are the same?

Reply:
Renamed the field to `placeholder`!